qulice-pmd/src/test/resources/com/qulice/pmd/StaticAccessToStaticFields.java

```diff
@@ -3,11 +3,15 @@
 public final class StaticAccessToStaticFields {
     private static int num = 1;
 
-    public static int number() {
+    private static int number() {
         return StaticAccessToStaticFields.num;
     }
 
     public int another() {
         return 0;
     }
+
+    public int addToNum(final int another) {
```

Comment:
@HDouss If to remove this method - we will get ` Avoid unused private methods such as 'number()'. (UnusedPrivateMethod)` violation